prusti-viper/src/encoder/mir/procedures/encoder/mod.rs

```diff
@@ -1290,237 +1290,217 @@ impl<'p, 'v: 'p, 'tcx: 'v> ProcedureEncoder<'p, 'v, 'tcx> {
         cleanup: &Option<mir::BasicBlock>,
```

Comment:
This function looks very large. Could you please move the builtin stuff to a separate file? (Similarly to `lifetimes.rs`.)